src/components/calcite-link/calcite-link.scss

```diff
@@ -29,6 +29,8 @@
 }
 
 .calcite-link--icon {
+  @apply relative;
+  top: 3px;
```

Comment:
Can we using an automatic way to center the icon instead of this hard coded value? What if the link is nested inside a heading element, won't it need to accommodate that different height?
```scss
@apply align-middle transition-default;
```

Reply:
Thanks Cari!

Reply:
Using align-middle doesn't seem to be vertically centered with the text
![image](https://user-images.githubusercontent.com/6144462/120513939-9ac91900-c381-11eb-9228-4e3e34c55790.png)

Is there another way to nudge it up a bit from where it is now?

Reply:
yeah when comparing that to button or anywhere else we use icons with text, this one is an outlier.